### ISSUE TITLE
Error: Trying to access array offset on value of type bool at PageController.php#407

### ISSUE THREAD

**Issue by mrclschstr** [eneiluj/cospend-nc#123]
I'm using PayForMe and connected the app with a Cospend project using the new [Share Links](https://github.com/eneiluj/cospend-nc/issues/9) with participant permissions. While using the app, my Nextcloud log gets flooded with the following errors:
```
[PHP] Error: Trying to access array offset on value of type bool at /<path>/nextcloud/apps/cospend/lib/Controller/PageController.php#407
GET /index.php/apps/cospend/api/projects/<id>/<password>/members
```

Full trace:
```
/<path>/nextcloud/apps/cospend/lib/Controller/PageController.php - line 407:
OC\Log\ErrorHandler::onError()

/<path>/nextcloud/apps/cospend/lib/Controller/PageController.php - line 1031:
OCA\Cospend\Controller\PageController->checkLogin()

/<path>/nextcloud/lib/private/AppFramework/Http/Dispatcher.php - line 217:
OCA\Cospend\Controller\PageController->apiGetMembers()

/<path>/nextcloud/lib/private/AppFramework/Http/Dispatcher.php - line 126:
OC\AppFramework\Http\Dispatcher->executeController()

/<path>/nextcloud/lib/private/AppFramework/App.php - line 156:
OC\AppFramework\Http\Dispatcher->dispatch()

/<path>/nextcloud/lib/private/Route/Router.php - line 302:
OC\AppFramework\App::main()

/<path>/nextcloud/lib/base.php - line 1006:
OC\Route\Router->match()

/<path>/nextcloud/index.php - line 36:
OC::handleRequest()
```

Is this really an error? What can I do about it?

My setup:
- Nextcloud: 22.2.5 (PHP 8.0.15)
- Cospend: 1.4.3
- PayForMe: 2.3.4

**Comment by justinlimbo** [comment]
I have exactly the same error messages flooding my NC log.

My setup of seems pretty much the same as @mrclschstr 
Nextcloud 22.2.5 on Ubuntu 20.04 (PHP 7.4.3,  mysql 8.0.28)
Cospend: 1.4.3
PayForMe 2.3.4 used by four different users/iPhones

**Comment by eneiluj** [comment]
Thanks for the issue.
It is fixed with v1.4.4 which was released a few minutes ago. Could you confirm?

**Comment by mrclschstr** [comment]
Works for me, no more errors in the log. Thank you!
